client/reader/following-manage/index.jsx

```diff
@@ -198,7 +204,7 @@ class FollowingManage extends Component {
 							/>
 						</div> }
 				</div>
-				{ ! sitesQuery && <RecommendedSites sites={ take( recommendedSites, 2 ) } /> }
+				{ hasFollows && ! sitesQuery && <RecommendedSites sites={ take( recommendedSites, 2 ) } /> }
```

Comment:
It would indeed, but I wasn't sure what the recommendations would be like if the user had no follows already. What are they driven off?